clang/lib/AST/TypePrinter.cpp

```diff
@@ -1909,6 +1911,12 @@ void TypePrinter::printAttributedAfter(const AttributedType *T,
     OS << " [[clang::lifetimebound]]";
     return;
   }
+  if (T->getAttrKind() == attr::LifetimeCaptureBy) {
+    // FIXME: Print the attribute arguments once we have a way to retrieve these
```

Comment:
Is it possible to get the `LifetimeCaptureByAttr` object from `T`?

Reply:
I think it should be possible since https://github.com/llvm/llvm-project/pull/108631 was merged.

Reply:
> I think it should be possible since #108631 was merged.
Thanks. Added.